releasenotes/notes/relax_wire_order_restrictions-ffc0cfeacd7b8d4b.yaml

```diff
@@ -0,0 +1,34 @@
+---
+features:
+  - |
+    Some restrictions when using ``wire_order`` in the circuit drawers had been relaxed.
+    Now, ``wire_order`` can list just qubits and, in that case, it can be used
+    with ``cregbundle=True``, since it will not affect the classical bits.
+
+    .. code-block::
+
+      from qiskit import QuantumCircuit, QuantumRegister, ClassicalRegister
+
+      qr = QuantumRegister(4, "q")
+      cr = ClassicalRegister(4, "c")
+      cr2 = ClassicalRegister(2, "ca")
+      circuit = QuantumCircuit(qr, cr, cr2)
+      circuit.h(0)
+      circuit.h(3)
+      circuit.x(1)
+      circuit.x(3).c_if(cr, 10)
+      circuit.draw('text', wire_order=[2, 3], cregbundle=True)
+
+    .. parsed-literal::
+
+       q_2: ────────────
+            ┌───┐ ┌───┐
+       q_3: ┤ H ├─┤ X ├─
+            ├───┤ └─╥─┘
+       q_0: ┤ H ├───╫───
+            ├───┤   ║
+       q_1: ┤ X ├───╫───
+            └───┘┌──╨──┐
+       c: 4/═════╡ 0xa ╞
+                 └─────┘
+      ca: 2/════════════
```

Comment:
I don't think this example will work after the new changes any more because of the partial wire order.  We should replace it with one that assigns all qubits.

Reply:
oops... fixed in d2224cdb093515f5eaad1f06c8f5cb72d0c6eff5